### ISSUE TITLE
Note to afternetwork.

### ISSUE THREAD

**Issue by smileycreations15** [AfterNetwork-smileycreations-Projects/Smile.js#87]
Hello @AfterNetwork. Sorry if I caused distruption in Smile.js but here it is why:
You demoted me too many times. I will take full control of the project, move it to my personal account, and contribute with forks

**Comment by AfterNetwork** [comment]
No. Please no. This is not acceptable. I will spam you everywhere. 
Look. Time for Github message.

personally, I put in the license, so hmmmm it’s under my license

**Comment by AfterNetwork** [comment]
Well if this is real, it’s uncapptle
